<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdentityDialog</class>
 <widget class="QWidget" name="IdentityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Identity</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="KComboBox" name="identitySelector">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAlphabetically</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="KPushButton" name="addIdentityButton"/>
     </item>
     <item row="0" column="3">
      <widget class="KPushButton" name="editIdentityButton"/>
     </item>
     <item row="0" column="4">
      <widget class="KPushButton" name="deleteIdentityButton"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Identities:</string>
       </property>
       <property name="buddy">
        <cstring>identitySelector</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="5">
      <widget class="KTabWidget" name="ktabwidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0" colspan="2">
            <widget class="KEditListBox" name="nickList">
             <property name="title">
              <string>Nicknames</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Real name:</string>
             </property>
             <property name="buddy">
              <cstring>realName</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="KLineEdit" name="realName"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Away</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0" colspan="2">
            <widget class="QCheckBox" name="markLastPosition">
             <property name="text">
              <string>Mark last position when away</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Away nickname:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>awayNickname</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="KLineEdit" name="awayNickname"/>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QGroupBox" name="useAwayMessages">
             <property name="title">
              <string>Away Messages</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Away message:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="buddy">
                   <cstring>awayMessage</cstring>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Return message:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="buddy">
                   <cstring>returnMessage</cstring>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="KLineEdit" name="awayMessage"/>
                </item>
                <item row="1" column="1">
                 <widget class="KLineEdit" name="returnMessage"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Misc</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Message Reasons</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_11">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Kick message:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="buddy">
                   <cstring>kickMessage</cstring>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Part message:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="buddy">
                   <cstring>partMessage</cstring>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Quit message:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="buddy">
                   <cstring>quitMessage</cstring>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="KLineEdit" name="kickMessage"/>
                </item>
                <item row="1" column="1">
                 <widget class="KLineEdit" name="partMessage"/>
                </item>
                <item row="2" column="1">
                 <widget class="KLineEdit" name="quitMessage"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
